.github/workflows/zulip-tests.yml

```diff
@@ -0,0 +1,62 @@
+name: Build
```

Comment:
Lowercase "build"; should look like this: https://img.shields.io/circleci/build/github/zulip/zulip/master.svg

Reply:
Okay. Will make it.